### PR TITLE
Changes to the Readme.md.j2 :

### PR DESCRIPTION
**Pull request by Salamandar**
* Fix whitespace / empty lines in generated files
* Add an 'upstream.version' variable (with fallback on 'manifest.version')
* Use manifest.name instead of manifest.id in some places

### CODE REVIEW

File: tools/README-generator/templates/README_fr.md.j2 [hunk 1]
```diff
@@ -1,42 +1,46 @@
...
-{% if upstream.userdoc %}* Documentation officielle utilisateur: {{ upstream.userdoc }}{% endif %}
-{% if upstream.admindoc %}* Documentation officielle de l'admin: {{ upstream.admindoc }}{% endif %}
-{% if upstream.code %}* Dépôt de code officiel de l'app:  {{ upstream.code }}{% endif %}
+{% if upstream.website -%}* Site official de l'app : {{ upstream.website }}
+{% endif -%}
+{% if upstream.userdoc -%}* Documentation officielle utilisateur: {{ upstream.userdoc }}
+{% endif -%}
+{% if upstream.admindoc -%}* Documentation officielle de l'admin: {{ upstream.admindoc }}
+{% endif -%}
+{% if upstream.code -%}* Dépôt de code officiel de l'app:  {{ upstream.code }}
```
Comment (ericgaspar) [inline tools/README-generator/templates/README_fr.md.j2:42]:
```suggestion
{% if upstream.userdoc -%}* Documentation officielle utilisateur : {{ upstream.userdoc }}
{% endif -%}
{% if upstream.admindoc -%}* Documentation officielle de l'admin : {{ upstream.admindoc }}
{% endif -%}
{% if upstream.code -%}* Dépôt de code officiel de l'app :  {{ upstream.code }}
```

File: tools/README-generator/templates/README_fr.md.j2 [hunk 2]
```diff
@@ -1,42 +1,46 @@
...
+{% endif -%}
 * Documentation YunoHost pour cette app: https://yunohost.org/app_{{manifest.id}}
```
Comment (ericgaspar) [inline tools/README-generator/templates/README_fr.md.j2:44]:
```suggestion
* Documentation YunoHost pour cette app : https://yunohost.org/app_{{manifest.id}}
```

File: tools/README-generator/templates/README_fr.md.j2 [hunk 3]
```diff
@@ -1,42 +1,46 @@
 # {{manifest.name}} pour YunoHost
 
-[![Niveau d'intégration](https://dash.yunohost.org/integration/{{manifest.id}}.svg)](https://dash.yunohost.org/appci/app/{{manifest.id}}) ![](https://ci-apps.yunohost.org/ci/badges/{{manifest.id}}.status.svg)  ![](https://ci-apps.yunohost.org/ci/badges/{{manifest.id}}.maintain.svg)
-[![Installer {{manifest.id}} avec YunoHost](https://install-app.yunohost.org/install-with-yunohost.svg)](https://install-app.yunohost.org/?app={{manifest.id}})
+[![Niveau d'intégration](https://dash.yunohost.org/integration/{{manifest.id}}.svg)](https://dash.yunohost.org/appci/app/{{manifest.id}}) ![](https://ci-apps.yunohost.org/ci/badges/{{manifest.id}}.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/{{manifest.id}}.maintain.svg)
```
Comment (ericgaspar) [inline tools/README-generator/templates/README_fr.md.j2:3]:
```suggestion
[![Niveau d'intégration](https://dash.yunohost.org/integration/{{manifest.id}}.svg)](https://dash.yunohost.org/appci/app/{{manifest.id}}) ![](https://ci-apps.yunohost.org/ci/badges/{{manifest.id}}.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/{{manifest.id}}.maintain.svg)  
```

File: tools/README-generator/templates/README_fr.md.j2 [hunk 4]
```diff
@@ -1,42 +1,46 @@
...
 If you don't have YunoHost, please consult [the guide](https://yunohost.org/#/install) to learn how to install it.*
 
 ## Vue d'ensemble
 
 {{manifest.description[lang]}}
 
-**Version incluse:** {{manifest.version}}
+**Version incluse:** {% if upstream.version %}{{upstream.version}}{% else %}{{manifest.version}}{% endif %}
 
 {% if upstream.demo %}**Démo:** {{upstream.demo}}{% endif %}
```
Comment (ericgaspar) [inline tools/README-generator/templates/README_fr.md.j2:18]:
```suggestion
{% if upstream.demo %}**Démo :** {{upstream.demo}}{% endif %}
```

**Review by ericgaspar — CHANGES_REQUESTED** [review]
Double space for line break before install badge and spaces ...

File: tools/README-generator/templates/README_fr.md.j2 [hunk 5]
```diff
@@ -1,43 +1,47 @@
...
+{% endif -%}
 
 ## Documentations et ressources
 
-{% if upstream.website %}* Site official de l'app : {{ upstream.website }}{% endif %}
-{% if upstream.userdoc %}* Documentation officielle utilisateur: {{ upstream.userdoc }}{% endif %}
-{% if upstream.admindoc %}* Documentation officielle de l'admin: {{ upstream.admindoc }}{% endif %}
-{% if upstream.code %}* Dépôt de code officiel de l'app:  {{ upstream.code }}{% endif %}
-* Documentation YunoHost pour cette app: https://yunohost.org/app_{{manifest.id}}
+{% if upstream.website -%}* Site official de l'app : {{ upstream.website }}
```
Comment (alexAubin) [inline tools/README-generator/templates/README_fr.md.j2:36]:
```suggestion
{% if upstream.website -%}* Site officiel de l'app : {{ upstream.website }}
```

Comment (Salamandar) [reply tools/README-generator/templates/README_fr.md.j2:36]: I don't understand, I commited this change already o.O

File: tools/README-generator/templates/README.md.j2 [hunk 6]
```diff
@@ -19,42 +19,46 @@ It shall NOT be edited by hand.
 
 # {{manifest.name}} for YunoHost
 
-[![Integration level](https://dash.yunohost.org/integration/{{manifest.id}}.svg)](https://dash.yunohost.org/appci/app/{{manifest.id}}) ![](https://ci-apps.yunohost.org/ci/badges/{{manifest.id}}.status.svg)  ![](https://ci-apps.yunohost.org/ci/badges/{{manifest.id}}.maintain.svg)
-[![Install {{manifest.id}} with YunoHost](https://install-app.yunohost.org/install-with-yunohost.svg)](https://install-app.yunohost.org/?app={{manifest.id}})
+[![Integration level](https://dash.yunohost.org/integration/{{manifest.id}}.svg)](https://dash.yunohost.org/appci/app/{{manifest.id}}) ![](https://ci-apps.yunohost.org/ci/badges/{{manifest.id}}.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/{{manifest.id}}.maintain.svg)
```
Comment (ericgaspar) [inline tools/README-generator/templates/README.md.j2:22]:
```suggestion
[![Integration level](https://dash.yunohost.org/integration/{{manifest.id}}.svg)](https://dash.yunohost.org/appci/app/{{manifest.id}}) ![](https://ci-apps.yunohost.org/ci/badges/{{manifest.id}}.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/{{manifest.id}}.maintain.svg)  
```

Comment (ericgaspar) [reply tools/README-generator/templates/README.md.j2:22]: Add line break for before install badge

**Review by ericgaspar — CHANGES_REQUESTED** [review]

File: tools/README-generator/templates/README.md.j2 [hunk 7]
```diff
@@ -8,8 +8,8 @@
 - Add a `LICENSE` file for the package.
 - Edit `doc/DISCLAIMER*.md`
 - The `README.md` files are to be automatically generated by https://github.com/YunoHost/apps/tree/master/tools/README-generator
-{% endif %}
 
+{% endif -%}
 ---
```
Comment (alexAubin) [inline tools/README-generator/templates/README.md.j2:13]:
```suggestion
---
{% endif -%}
```

**Comment by Salamandar** [comment]
:+1:

**Pull request merged**